### PR TITLE
Update config guide - Closes #510

### LINKED ISSUE #510: Update configure guide
Parent issue: #505

### PR DESCRIPTION
**Pull request by Tschakki**
Closes #510

### CODE REVIEW

File: modules/ROOT/pages/guides/configuration.adoc [hunk 1]
```diff
@@ -5,108 +5,97 @@ Mona Bärenfänger <mona@lightcurve.io>
...
 
 :url_reference_config: reference/config.adoc
 
-[[modify_default_values]]
-== Modify default values
-If you wish to modify the default values, the following paragraph explains how to perform this, and also provides a devnet configuration example.
-A devnet configuration can be thought of as a local network on your machine, whereby it is possible to safely experiment with the Lisk SDK.
-The following configuration example shown here is a pre-configured default configuration to set up an instance.
+[[intro]]
+The `Application` instance, which is exported by the Lisk SDK, is configured to run with the <<config_options, default config options>> that are defined in the Lisk Framework.
```
Comment (chris529) [inline modules/ROOT/pages/guides/configuration.adoc]:
```suggestion
The `Application` instance which is exported by the Lisk SDK, is configured to run with the <<config_options, default config options>> that are defined in the Lisk Framework.
```

File: modules/ROOT/pages/guides/configuration.adoc [hunk 2]
```diff
@@ -5,108 +5,97 @@ Mona Bärenfänger <mona@lightcurve.io>
...
+    });
 ----
 
-<1> Require the lisk-sdk package.
-<2> Passes the config json `configDevnet` as config object for the new application instance.
-This will start the node with a fully functioning Devnet.
-
-=== Overwrite the devnet example config
+<1> Imports the `configDevnet` object.
+<2> Optionally override certain options in `configDevnet`.
```
Comment (chris529) [inline modules/ROOT/pages/guides/configuration.adoc]:
```suggestion
<2> Optionally overrides certain options in `configDevnet`.
```

File: modules/ROOT/pages/guides/configuration.adoc [hunk 3]
```diff
@@ -5,108 +5,97 @@ Mona Bärenfänger <mona@lightcurve.io>
...
+<3> Create the application instance and pass the `genesisBlockDevnet` as genesis block and `configDevnet` as config.
```
Comment (chris529) [inline modules/ROOT/pages/guides/configuration.adoc]:
```suggestion
<3> Creates the application instance and passes the `genesisBlockDevnet` as genesis block and `configDevnet` as config.
```

File: modules/ROOT/pages/guides/configuration.adoc [hunk 4]
```diff
@@ -5,108 +5,97 @@ Mona Bärenfänger <mona@lightcurve.io>
...
-The configuration object `configDevnet` possesses the structure as shown in the xref:{url_reference_config}[list of configuration options].
-It can be modified accordingly, before passing the config to the `Application` instance as shown below:
+[[config_options]]
+== Configuration options
 
-.Example how to override the example config object:
-[source,js]
-----
-const { Application, genesisBlockDevnet, configDevnet} = require('lisk-sdk'); <1>
+The default configuration options are located in the {url_github_sdk}[Lisk SDK repository] on GitHub:
```
Comment (chris529) [inline modules/ROOT/pages/guides/configuration.adoc]:
```suggestion
The default configuration options are located in the {url_github_sdk}[Lisk SDK repository] in GitHub:
```

File: modules/ROOT/pages/guides/configuration.adoc [hunk 5]
```diff
@@ -5,108 +5,97 @@ Mona Bärenfänger <mona@lightcurve.io>
...
+====
+Please do not change the default values in these files directly as they will be overwritten when software updates are performed.
+Instead of changing the default values, define the custom configuration options inside your blockchain application.
+====
 
 === Constants
 
-Inside `app.genesisConfig` specific constants for the blockchain application are set.
+Inside `configDevnet.app.genesisConfig` specific constants for the blockchain application are set.
+These constants allow to quickly adjust basic properties of the blockchain, for example time that elapses until a new block is forged, the network epoch time, or the block rewards.
```
Comment (chris529) [inline modules/ROOT/pages/guides/configuration.adoc]:
```suggestion
These constants allow the basic properties of the blockchain to be easily adjusted. For example, the time that elapses until a new block is forged, the network epoch time, and the block rewards.
```

File: modules/ROOT/pages/guides/configuration.adoc [hunk 6]
```diff
@@ -5,108 +5,97 @@ Mona Bärenfänger <mona@lightcurve.io>
...
 
-In the alpha version of the Lisk SDK, not all available constants are configurable by the user.
-In future versions of the Lisk SDK, more constants will become configurable.
+In the alpha version of the Lisk SDK, not all available constants are configurable by the user, yet.
```
Comment (chris529) [inline modules/ROOT/pages/guides/configuration.adoc]:
```suggestion
In the alpha version of the Lisk SDK, at present not all of the available constants are configurable by the user yet.
```

File: modules/ROOT/pages/guides/configuration.adoc [hunk 7]
```diff
@@ -5,108 +5,97 @@ Mona Bärenfänger <mona@lightcurve.io>
...
+In future versions, more constants will become configurable.
```
Comment (chris529) [inline modules/ROOT/pages/guides/configuration.adoc]:
```suggestion
However, in future versions more constants will become configurable.
```

File: modules/ROOT/pages/guides/configuration.adoc [hunk 8]
```diff
@@ -5,108 +5,97 @@ Mona Bärenfänger <mona@lightcurve.io>
...
 The genesis block describes the very first block in the blockchain.
 It defines the initial state of the blockchain on start of the network.
 
 The genesis block is not forged by a delegate, such as all of the other blocks which come after the genesis block.
-Instead it is defined by the blockchain application developer, when creating the `Application` instance of the blockchain app, (see section <<modify_default_values,Modify default values>>).
+Instead it is defined by the developer, when <<intro,creating the Application>> instance of the blockchain application.
+
+=== The devnet genesis block
+
+The Lisk SDK exposes an object `genesisBlockDevnet` that holds all important information to spin up a local development network.
```
Comment (chris529) [inline modules/ROOT/pages/guides/configuration.adoc]:
```suggestion
The Lisk SDK exposes an object `genesisBlockDevnet` that holds all of the required important information to spin up a local development network.
```

**Review by chris529 — CHANGES_REQUESTED** [review]
Hi Mona. looks very good...just a few minor points  :-)

File: modules/ROOT/pages/guides/configuration.adoc (hunk 2 above)
Comment (Tschakki) [reply modules/ROOT/pages/guides/configuration.adoc]: Maybe:
```suggestion
<2> Optionally it is possible to override certain options in `configDevnet` before creating the application instance.
```

**Review by chris529 — APPROVED** [review]
Perfect :-)

**Pull request merged**